internal/pkg/errors/error_message.go

```diff
@@ -62,6 +62,7 @@ const (
 	EnvNotFoundErrorMsg    = `environment "%s" not found`
 	EnvNotFoundSuggestions = "List available environments with `confluent environment list`."
 	EnvSwitchErrorMsg      = "failed to switch environment: failed to save config"
+	UseExistingEnvMsg      = `Switch to an active environment with "confluent environment use"`
```

Comment:
Suggestion: `Select a new active environment` instead of `Switch to an active environment`

Reply:
```suggestion
	UseExistingEnvMsg      = "Select a new environment with `confluent environment use`."
```

Reply:
I'd also drop the "active", the point of selecting an environment is to make it active.